packages/middleware/src/entries.js

```diff
@@ -10,6 +10,15 @@ _ENTRIES['middleware_pages/_middleware'] = {
         response: result,
       };
     }
-    return result;
+
```

Comment:
What about if the user does `return null`? Do we handle that differently than `return undefined`?

Reply:
The platform always expects a result like `{ waitUntil, response, promise }` I'm not aware of any special convention for returning null from a middleware.